File: niftyn8+elixir/euler1.ex
defmodule Euler1 do
  def solve(num) do
    0..num
    |> Enum.filter( &(rem(&1, 3) == 0 || rem(&1, 5) == 0 ) )
    |> :lists.sum
  end
end

IO.puts("Answer is: #{Euler1.solve(1000)}")

File: niftyn8+ruby/euler1.rb
find_sum = ->(num) do
  Array(1..num).select { |num| num % 3 == 0 || num % 5 == 0 }
               .reduce(0) { |sum, num| sum + num }
end

puts "Answer is: #{find_sum.call(1000)}"
